### PR TITLE
Adding new blog post

### PR DESCRIPTION
**Pull request by EbonyLouis**
Adding new blog post on who really owns your social media handles and how DIDs are a solution

### CODE REVIEW

**Review by ALRubinger — CHANGES_REQUESTED** [review]
Please review extra dep as I'm noting in line changes?

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "test": "pnpm vitest --config vite.config.ts"
   },
   "dependencies": {
-    "@tbd54566975/web5": "0.7.11"
+    "@tbd54566975/web5": "0.7.11",
+    "react-markdown": "^8.0.7"
```
Comment (ALRubinger) [inline package.json:14]: Should this new dependency be added for the new blog post? Seems off in this PR, no?

**Review by ALRubinger — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (EbonyLouis) [reply package.json:14]: when I was trying to get it to compile locally, It kept saying it couldn't resolve 'react-markdown' so I had to npm install it. I can remove it.

Comment (dayhaysoos) [reply package.json:14]: Pretty sure this is due to mic's changes for the ChatGPT plugin

**Review by angiejones — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (angiejones) [reply package.json:14]: so should this line be removed?

Comment (dayhaysoos) [reply package.json:14]: @angiejones she just recently pulled in changes from main so I think it's fine now.

Comment (ALRubinger) [reply package.json:14]: Yeah even if it's needed on the dev site, should be in the deps for the `site` workspace and not top-level. Sounds sorted now; thanks all.

Comment (ALRubinger) [reply package.json:14]: Rather, I guess it'll be sorted once this line is off the diffs on this PR :)

Comment (angiejones) [reply package.json:14]: confirming that this is already in main
https://github.com/TBD54566975/developer.tbd.website/blame/main/site/package.json#L38

**Review by angiejones — APPROVED** [review]

**Pull request merged**